Add tests for model mixin

KengoWada opened the pull request:
Resolves #25 

## Added tests for TypesenseModelMixin

###  Changes made

- Added test settings
- Added test models and collections
- Added Django test runner
- Fixed `TypesenseCollection` delete method to convert object ids to `int`
- Updated `CONTRIBUTIONS.md` to reflect changes in tests.
- Updated GitHub actions to include TypeSense

EricOuma commented:
Some of the skeleton code I added for the tests are slightly obsolete. The tests you have added are actually for `test_typesense_collection` and not `test_typesense_model_mixin`

KengoWada commented:
> Some of the skeleton code I added for the tests are slightly obsolete. The tests you have added are actually for `test_typesense_collection` and not `test_typesense_model_mixin`

Hey @EricOuma noted. I've updated the tests to target `django_typesense.mixins`
